### PR TITLE
round target point coordinates

### PR DESCRIPTION
**Pull request by bkrmendy**
## Problem:
`dragging from the sidebar works` from the `image dnd` test suite is very flaky due to coordinate rounding errors

## Fix:
Use rounded coordinates for the drop coordinate

#### Bonus change
`<KeyIndicator />` had an unnecessarily assigned `key` prop, removed it after being pestered for it by @gbalint

### CODE REVIEW

File: editor/src/components/canvas/controls/select-mode/canvas-strategy-picker.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const CanvasStrategyPicker = React.memo(() => {
                     opacity: isStrategyFailure && strategy.id === activeStrategy ? 0.5 : 1,
                   }}
                 >
-                  <KeyIndicator key={index + 1} keyNumber={index + 1} />
+                  <KeyIndicator keyNumber={index + 1} />
```
Comment (balazsbajorics) [inline editor/src/components/canvas/controls/select-mode/canvas-strategy-picker.tsx:113]: sigh, unrelated changes are the worst!

File: editor/src/templates/image-dnd.spec.browser2.tsx [hunk 2]
```diff
@@ -233,17 +233,17 @@ describe('image dnd', () => {
...
-    })
+    }
 
     const target = editor.renderedDOM.getByTestId('scene')
     const targetBounds = target.getBoundingClientRect()
 
     const endPoint = {
-      x: targetBounds.x + targetBounds.width / 2,
-      y: targetBounds.y + targetBounds.height / 2,
+      x: (targetBounds.x + targetBounds.width / 2) ^ 0,
```
Comment (balazsbajorics) [inline editor/src/templates/image-dnd.spec.browser2.tsx:245]: I think Math.round or Math.floor would be much more readable here.

Comment (bkrmendy) [reply editor/src/templates/image-dnd.spec.browser2.tsx]: fixed

**Review by Rheeseyb — APPROVED** [review]

**Review by enidemi — APPROVED** [review]

**Review by seanparsons — APPROVED** [review]
